static_src/components/activity_log.jsx

```diff
@@ -0,0 +1,69 @@
+
+import style from 'cloudgov-style/css/cloudgov-style.css';
+import React from 'react';
+
+import ActivityLogItem from './activity_log_item.jsx';
+import ActivityStore from '../stores/activity_store';
+import createStyler from '../util/create_styler';
+
+function stateSetter(props) {
+  const activity = ActivityStore.getAll().filter((item) => {
+    if (item.type === 'audit.service_binding.create') {
+      return item.metadata.request.app_guid === props.initialAppGuid;
+    }
+    return item.actee === props.initialAppGuid;
+  }).sort((a, b) => new Date(b.timestamp) - new Date(a.timestamp));
+
+  return {
+    activity
+  };
+}
+
+const propTypes = {
+  initialAppGuid: React.PropTypes.string.isRequired,
+  title: React.PropTypes.string
+};
+
+const defaultProps = {
+  title: 'Activity Log'
+};
+
+export default class ActivityLog extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = stateSetter(props);
+    this.styler = createStyler(style);
+
+    this._onChange = this._onChange.bind(this);
+  }
+
+  componentDidMount() {
+    ActivityStore.addChangeListener(this._onChange);
+  }
+
+  componentWillUnmount() {
+    ActivityStore.removeChangeListener(this._onChange);
+  }
+
+  _onChange() {
+    this.setState(stateSetter(this.props));
+  }
+
+  render() {
+    return (
+      <div className={ this.styler('activity_log-container') }>
+        <h1>{ this.props.title }</h1>
```

Comment:
@msecret Do you think this should be an `<h1>` semantically? If not, maybe this should just be a `<strong>`? It is a heading, and it is a component so it might not be the only heading on the page. I think we can also change the wrapper `<div>` to be a `<section>` since each section can have its own heading hierarchy.